src/Util.cpp

```diff
@@ -569,7 +569,8 @@ IMAGE_TYPE utilFindType(const char *file, char (&buffer)[2048])
                 return IMAGE_UNKNOWN;
         }
         MultiByteToWideChar(CP_ACP, 0, file, -1, pwText, dwNum);
-        char *file_conv = fex_wide_to_path(pwText);
+        //char *file_conv = fex_wide_to_path(file);
+        char *file_conv = (char *)file;
```

Comment:
We already have a UTF-8 string here. The steps above seems unnecessary. Before, we had a multibyte string (UTF-16), that we get a `wchar_t *` from and then we convert into a `char *` UTF-8 string.

The key idea here is that the first string we have is `wxString`. We can use its functions to get the desired format. So when the code runs here, `const char *file` is already what we want(?).